playbooks/trust/add-trust.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Playbook to create a trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+    - name: create a one-way trust to the windows.local domain
```

Comment:
States are ensured by Ansible. Use "ensure trust is present".

Reply:
done